mod_project/layer.py

```
@@ -277,8 +512,50 @@ def _parse(self):
                     'gauge':      normalize_track_gauge(
                         v.get('gauge', v.get('Gauge', 'Standard'))
                     ),
+                    'bridgeSupportsSteel': bool(v.get('bridgeSupportsSteel', False)),
+                    'yard': bool(v.get('yard', False)),
                 }
 
+    def _merge_native_industries_into_areas(self):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_merge_native_industries_into_areas`

Add return type annotation: `None`

(ANN202)